src/hotspot/share/prims/jvmti.xml

```diff
@@ -1004,7 +1004,7 @@ jvmtiEnv *jvmti;
     In some cases, <jvmti/> functions allocate memory that your program must
     explicitly deallocate. This is indicated in the individual <jvmti/>
     function descriptions.  Empty lists, arrays, sequences, etc are
-    returned as <code>nullptr</code>.
+    returned as a null pointer (C <code>NULL</code> or C++ <code>nullptr</code>).
```

Comment:
This may be a little unnecessary rigor, but I believe that `nullptr` is not a null pointer. `nullptr` is the pointer literal that can be implicitly converted to a null pointer value of any pointer type and any pointer to member type. And I think the thing returned here is a null pointer, not `nullptr`.

Reply:
I'm not sure I understand this comment. Sorry.